Makefile

```diff
@@ -29,7 +29,7 @@ verilator      ?= verilator
 # traget option
 target-options ?=
 # additional definess
-defines        ?= WT_DCACHE+RVFI_TRACE
+defines        ?= WT_DCACHE+RVFI_TRACE+BITMANIP
```

Comment:
We are trying to not use tick defines for the inclusion of features (i.e., selecting rv32, rv64, A, M, etc. extensions). That is all done using parameters. I would suggest you add a `parameter` to the `ariane_pkg` called.

```
parameter bit BITMANIP = 1'b1;
```